src/scripts/main.js

```diff
@@ -1,5 +1,27 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  let x = e.clientX;
+  let y = e.clientY;
+  const spider = document.querySelector('.spider');
```

Comment:
Ensure that the element with the class 'spider' exists in the DOM before trying to access it. Consider adding a check to handle cases where it might be `null`.